docker/Dockerfile

```diff
@@ -19,14 +19,17 @@ FROM ubuntu:jammy AS base-runtime
 ARG AMD_ARCH=AMD_GFX90A
 ARG CUDA_ARCH=AMPERE80
 ARG CUDA_INSTALLER=https://developer.download.nvidia.com/compute/cuda/12.2.2/local_installers/cuda_12.2.2_535.104.05_linux.run
+ARG DEBIAN_FRONTEND=noninteractive
 ARG GCC_VERSION=11
 ARG KOKKOS_VERSION=4.2.00
-ARG LIGHTNING_VERSION=v0.34.0
+ARG PENNYLANE_VERSION=v0.36.0
+ARG LIGHTNING_VERSION=v0.36.0
```

Comment:
Is it better to fetch these from the environment?  Would anyone every want to build a very specific container by setting a PENNYLANE_VERSION env var and then running make?

Reply:
I'll repost my comment below

> We seldom use that Makefile, it is more like a cheat sheet for invoking docker build for me.

So I'd say, no need for an env var, let's just pass extra options with `--build-arg`.